Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +17,31 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            // To do: call your implementation.             
+            Console.WriteLine(message);
+            return int.Parse(Console.ReadLine());
         }
 
         public static string PromptString(string message)
         {
             // To do: call your implementation. 
-            return "";
+            Console.WriteLine(message);
+            return Console.ReadLine();
         }
 
         public static float PromptFloat(string message)
         {
             // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            return float.Parse(Console.ReadLine());
         }
 
         public static float CalculateBmi(float weight, float height)
         {
-            // To do: call your implementation. 
-            return 0;
+            // To do: call your implementation.
+            float mSquared = height * height;
+            float bmi = weight / mSquared;
```

Comment:
Should simply have multiplication and division in the same line, because they represent the same thing.